doc/developer_guide/HNS_error_bestiary.md

```diff
@@ -1,15 +1,120 @@
 # HNS errors bestiary
 
-HNS error messages ofter aren't descriptive enough. 
-Here are error messages the team encountered in different contexts, and their meaning.
+HNS error messages ofter aren't descriptive enough, and the cleanup is quite hard.
+The following document describes common cleanup techniques and error interpretations.
+
+## HNS Decontamination procedures
+
+Before working with HNS, one must realize the risks.
+There are many ways in which HNS can stop working and affect other parts of the system.
+For a developer, it is important to be able to recognize those, which can be recovered 
+from. Below are some techniques which may help.
+
+Always try cleaning using the lowest level procedure. The higher the level, the more 
```

Comment:
The lower lever procedures, ie. with the lower number are actually higher-level, ie. farther from the metal. This is somewhat confusing, but not sure how to reword it. This could be addressed later.

Reply:
Maybe reword it using greek letters? Level Alpha decontamination procedure

The highest level would be Level [REDACTED] decontamination procedure.

Thoughts?